### ISSUE TITLE
Expose typedefs to define global event handler function

### ISSUE THREAD

**Issue by tobz1000** [kitbagjs/events#19]
Hey - thanks for this lib, using it has been a good experience so far.

I would like to define the global handler for an emitter in a different location to its assignment, e.g.:

```ts
const onEvent = ({ kind, payload }: GlobalEvent<MyEvent>) => { ... };

// Elsewhere...
const myEvent = createEmitter<MyEvent>();
myEvent.on(onEvent);
```

Would it be feasible to expose the `GlobalEvent` type for this purpose? Or any other good way to achieve this?

**Comment by stackoverfloweth** [comment]
@tobz1000 glad you're enjoying it!

We will definitely give some thought to exporting some types. In the meantime, you definitely _could_ re-create the types

```ts
type MyGlobalEvent = {
  [K in keyof MyEvent]: {
    kind: K,
    payload: MyEvent[K],
  }
}[keyof MyEvent]

type MyGlobalEventHandler = (event: MyGlobalEvent) => void
```

which, thanks to duck typing, would satisfy your use case

```ts
const onEvent: MyGlobalEventHandler = ({ kind, payload }) => { ... };

// Elsewhere...
const myEvent = createEmitter<MyEvent>();
myEvent.on(onEvent);
```

**Comment by tobz1000** [comment]
Thanks for the quick reply - that's more or less what I've done for now. I'll keep an eye on future versions to see if a neater solution is available in future. :)

**Comment by pleek91** [comment]
@tobz1000 I just published 0.1.1 which exports both the `GlobalEventHandler` and `GlobalEvent` types. So you should be able to update and then do this now. 

```ts
const onEvent: GlobalEventHandler<MyEvents> = ({ kind, payload }) => { ... }
```
